### PR TITLE
[BUILD][LICENSE] Dist with LICENSE-binary

### PR DESCRIPTION
**Pull request by pan3793**
pan3793 [Closes #514](https://github.com/yaooqinn/kyuubi/pull/514)    Target Issue Test Plan [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: build/dist [hunk 1]
```diff
@@ -201,6 +201,7 @@ fi
 
 # Copy license and ASF files
 cp "$KYUUBI_HOME/LICENSE" "$DISTDIR"
+cp "$KYUUBI_HOME/LICENSE-binary" "$DISTDIR"
```
[the remark this answers is not in the document]
Comment (pan3793) [reply build/dist:204]: OK

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
thanks, merged to master for v1.2.0
